libcontainer/cgroups/fs/blkio.go

```diff
@@ -226,12 +230,15 @@ func getStats(path string, stats *cgroups.Stats) error {
 	if blkioStats, err = getBlkioStat(filepath.Join(path, "blkio.throttle.io_service_bytes")); err != nil {
 		return err
 	}
-	stats.BlkioStats.IoServiceBytesRecursive = blkioStats
+	//	stats.BlkioStats.IoServiceBytesRecursive = blkioStats
```

Comment:
I digged a bit and found the PR added this code: https://github.com/docker/libcontainer/pull/167

Looks like it's intentional.